src/Monolog/Processor/IntrospectionProcessor.php

```diff
@@ -55,6 +55,14 @@ public function __invoke(array $record)
             return $record;
         }
 
+        /*
+        * http://php.net/manual/en/function.debug-backtrace.php
+        * As of 5.3.6, DEBUG_BACKTRACE_IGNORE_ARGS option was added.
+        * Any version less than 5.3.6 must use the DEBUG_BACKTRACE_IGNORE_ARGS constant value '2'.
+        */
+        if (!defined('DEBUG_BACKTRACE_IGNORE_ARGS')) {
+            define('DEBUG_BACKTRACE_IGNORE_ARGS', 2);   
```

Comment:
I am not arguing against fixing compat for PHP < 5.3.6. I just would not define the constant here as other libraries may use the presence of this constant to conditionally execute some code. That's why I said I would check for the value of the `PHP_VERSION_ID` constant instead.